syft/serde/serde.py

```diff
@@ -123,6 +123,12 @@
 
 ## SECTION: High Level Simplification Router
 def _force_full_simplify(obj: object) -> object:
+    """To force a full simplify genrally if the usual _simplify is not suitable.
+    
+    Args:
+    :param: the object
```

Comment:
`:param:` should be `param:`. And the same applies to all other arguments and returns.